src/commands/database-rules-canary.ts

```diff
@@ -0,0 +1,23 @@
+import { Command } from "../command";
+import * as logger from "../logger";
+import * as requireInstance from "../requireInstance";
+import { requirePermissions } from "../requirePermissions";
+import * as metadata from "../database/metadata";
+
+export default new Command("database:rules:canary <ruleset_id>")
```

Comment:
We should stick to consistent casing for parameters; lowerCamelCase has generally been preferred

Reply:
lowerCamelCase it is :+1: